sql/7_0_0-to-7_0_1_upgrade.sql

```diff
@@ -197,3 +197,7 @@ CREATE UNIQUE INDEX `uuid` ON `openemr_postcalendar_events` (`uuid`);
 #IfNotRow2D list_options list_id drug_route option_id bymouth
 INSERT INTO `list_options` (`list_id`, `option_id`, `title`, `seq`, `is_default`, `notes`, `codes`) VALUES ('drug_route', 'bymouth', 'By Mouth', 1, 0, 'PO', 'NCI-CONCEPT-ID:C38288');
```

Comment:
Fix this so it is consistent with the rest of the list. 'bymouth' to 25. Save as integer not string.
We have other lists with similar id pattern i.e Drug Interval.

Simply making this change should fix the original issue. At least it did for me.

Reply:
problem with setting to `25` would be the possibility of clobbering somebody's already established list

Reply:
That's why you re-sequence lists in upgrade script using vars. There are several examples in past upgrades. Changing the data type for accommodate one new item is not the answer IMO.

Reply:
the other string items haven't been working all along as well

Reply:
Just do a query to get the max id value and add 1 to it then insert new item. If you want I can write one for you.

Reply:
Why make it more complicated than it has to be? I can see the alternative of simply ensuring all those values are converted over to integers, but why make more database modifications than necessary when the codebase changes are minimal?

Reply:
@sjpadgett's right about the implications of other folks using the prescription's table

and the sql max id thing is pretty cool in the upgrade script

Reply:
Let me mull over a better approach over lunch and get back to you - either way I didn't consider that case fully, and I'm starting to think you're correct. If the PR looks drastically different I'll close this one and open a new one.